pamelalozano/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +27,19 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+     
+     JSONArray ja = new JSONArray();
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Comment prueba = new Comment("Title", "Msg");
+     JSONObject jo = new JSONObject();
```

Comment:
coming up with a more descriptive name for "jo" and "ja" will help us understand what you're trying to do with them! I think the JSON array in servlet memory is meant to mimick a database for a later step.